### PR TITLE
fix(app): Fix protocol setup screen dqa3

### PR DESCRIPTION
**Pull request by koji**
# Overview
Fix protocol setup screen dqa3
close RAUT-498

I removed protocol setup labware part because of https://github.com/Opentrons/opentrons/pull/12980

protocol analysis failure modal
Screenshot 2023-06-26 at 4 35 08 PM

# Test Plan

# Changelog

# Review requests

# Risk assessment
low

### CODE REVIEW

**Comment by koji** [comment]
- [x] LocationIcon in Labware need to add a hub function to parse the return

File: app/src/organisms/ProtocolSetupLabware/index.tsx [hunk 1]
```diff
@@ -70,6 +74,13 @@ const LabwareThumbnail = styled.svg`
   flex-shrink: 0;
 `
 
+const MODULE_ICON_NAME_BY_TYPE: { [type in string]: IconName } = {
```
Comment (ewagoner) [inline app/src/organisms/ProtocolSetupLabware/index.tsx]: Can you leverage the same mapping in the ModuleIcon component so we don't have two of these to maintain?

Comment (koji) [reply app/src/organisms/ProtocolSetupLabware/index.tsx]: Swtiched to location.moduleId.

**Comment by koji** [comment]
I'm going to remove the changes for ProtocolSetup Labware since it is overlapping https://github.com/Opentrons/opentrons/pull/12980/files#top

File: app/src/molecules/LocationIcon/index.tsx [hunk 2]
```diff
@@ -30,7 +30,9 @@ export function LocationIcon({
       alignItems={ALIGN_CENTER}
       height="2rem"
       width="max-content"
-      padding={`${SPACING.spacing4} ${SPACING.spacing8}`}
+      padding={`${SPACING.spacing4}  ${
+        slotName != null ? SPACING.spacing8 : '6px'
```
Comment (ewagoner) [inline app/src/molecules/LocationIcon/index.tsx]: Should the 6px be 0.375rem?

**Review by ewagoner — APPROVED** [review]

File: app/src/molecules/LocationIcon/index.tsx (hunk 2 above)
Comment (koji) [reply app/src/molecules/LocationIcon/index.tsx]: oh good catch, thanks!

**Pull request merged**